### PR TITLE
[ADD] pos_invoice_paid_from_pos

### PR DESCRIPTION
**Pull request by carmenbianca**
Signed-off-by: Carmen Bianca Bakker <carmen@coopiteasy.be>

## Odoo task (if applicable)

https://gestion.coopiteasy.be/web#id=8882&action=475&active_id=231&model=project.task&view_type=form&menu_id=

## Checklist before approval

- [ ] Tests are present (or not needed).
- [x] Credits/copyright have been changed correctly.
- [x] Change log snippet is present.
- [ ] (If a new module) Moving this to OCA has been considered.

### CODE REVIEW

File: pos_invoice_paid_from_pos/views/report_invoice.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <template id="report_invoice_document" inherit_id="account.report_invoice_document">
+        <xpath expr="//p[@t-if='o.reference']" position="attributes">
```
Comment (victor-champonnois) [inline pos_invoice_paid_from_pos/views/report_invoice.xml:5]: Some conflicts could happen with this [template](https://github.com/coopiteasy/cie-custom/blob/9d6d4bf54c909d659e6f12a3c977085016cc7dd1/account_invoice_pos_already_paid/views/templates.xml#L12), maybe you could add a comment to signal it (preventively) ?

**Review by victor-champonnois — APPROVED** [review]
LGTM :) just one comment

File: pos_invoice_paid_from_pos/views/report_invoice.xml (hunk 1 above)
Comment (carmenbianca) [reply pos_invoice_paid_from_pos/views/report_invoice.xml:5]: Good idea, let's write some comments and documentation :)

**Pull request merged**